tox.ini

```diff
@@ -2,9 +2,9 @@
 min_version =  4.11
 
 env_list =
-    python{3.8,3.9,3.10}-dj{32}-wagtail{41,52}
-    python{3.11}-dj{41,42}-wagtail{52,60}
-    python{3.12}-dj{42,50}-wagtail{52,60}
+    python{3.8,3.9,3.10,3.11,3.12}-django3.2-wagtail5.2
+    python{3.8,3.9,3.10,3.11}-django4.2-wagtail{5.2,6.0}
+    python{3.10,3.11,3.12}-django5.0-wagtail{5.2,6.0}
```

Comment:
```suggestion
    python{3.11,3.12}-django5.0-wagtail{5.2,6.0}
```

Reply:
Hi @zerolab ,
Thanks for the suggestions. Applied those here:
https://github.com/torchbox/wagtail-headless-preview/pull/62/commits/4af7d7bf65694b7ce603e39c81cc746d8cfdd4e3